### PR TITLE
[5.1] Added public function getCustomDirectives to BladeCompiler

### PR DESCRIPTION
**Pull request by ihoru**
I need to get all custom directives in this component:
https://github.com/xinax/laravel-gettext

More details here:
https://github.com/xinax/laravel-gettext/issues/34

### CODE REVIEW

File: src/Illuminate/View/Compilers/BladeCompiler.php [hunk 1]
```diff
@@ -741,6 +741,16 @@ public function directive($name, callable $handler)
     }
 
     /**
+     * Get the list of custom directives.gs
```
Comment (GrahamCampbell) [inline src/Illuminate/View/Compilers/BladeCompiler.php]: Please drop the `gs`.

File: tests/View/ViewBladeCompilerTest.php [hunk 2]
```diff
@@ -487,9 +487,11 @@ public function testCustomExtensionsAreCompiled()
     public function testCustomStatements()
     {
         $compiler = new BladeCompiler($this->getFiles(), __DIR__);
+        $this->assertEquals(0, count($compiler->getCustomDirectives()));
```
Comment (GrahamCampbell) [inline tests/View/ViewBladeCompilerTest.php]: Please use assertCount.

File: tests/View/ViewBladeCompilerTest.php [hunk 3]
```diff
@@ -487,9 +487,11 @@ public function testCustomExtensionsAreCompiled()
...
         $compiler->directive('customControl', function ($expression) {
             return "<?php echo custom_control{$expression}; ?>";
         });
+        $this->assertEquals(1, count($compiler->getCustomDirectives()));
```
Comment (GrahamCampbell) [inline tests/View/ViewBladeCompilerTest.php]: same

File: tests/View/ViewBladeCompilerTest.php (hunk 2 above)
Comment (ihoru) [inline tests/View/ViewBladeCompilerTest.php]: Thanks

File: src/Illuminate/View/Compilers/BladeCompiler.php (hunk 1 above)
Comment (ihoru) [inline src/Illuminate/View/Compilers/BladeCompiler.php]: Oh, no! What my "git status" alias is doing here? :)

**Pull request merged**